src/Nethermind/Nethermind.Consensus/ExecutionRequests/ExecutionRequestProcessor.cs

```diff
@@ -73,7 +74,7 @@ public void ProcessDeposits(TxReceipt[] receipts, IReleaseSpec spec, ArrayPoolLi
                 for (var j = 0; j < logEntries.Length; j++)
                 {
                     LogEntry log = logEntries[j];
-                    if (log.Address == spec.DepositContractAddress)
+                    if (log.Address == spec.DepositContractAddress && log.Topics.Length >= 1 && log.Topics[0] == DepositEventAbi.Hash)
```

Comment:
Let's use hardcoded version

Reply:
We are already using ABI for parsing, this is actually safer.